docs/animations/AnimationSet.md

```diff
@@ -0,0 +1,103 @@
+# AnimationSet
+
+The AnimationSet class defines an object for storing and managing CompositionAnimations for an element. AnimationSet includes Blur, Fade, Light, Offset, Rotate, Saturation and Scale animations. AnimationSet animations is applied to all the XAML elements in its parent control/panel. AnimationSet animations doesn't affect the functionality of the control.
+
+## Syntax
+
+**XAML**
+
+```xml
```

Comment:
The Xaml code doesn't really have anything to do with AnimationSet, AnimationSet is meant for c#